test/test_utils/__init__.py

```diff
@@ -35,6 +35,8 @@
 DEFAULT_REGION = "us-west-2"
 # Constant to represent region where p3dn tests can be run
 P3DN_REGION = "us-east-1"
+# Constant to represent region where p4de tests can be run
+P4DE_REGION = "us-east-1"
```

Comment:
is this temporary?

Reply:
No, this is a permanent code-change, which can later be modifed if we decide to change the region in which we launch p4de instances.